plugin.json

```
@@ -81,6 +81,13 @@
                 "placeholder": "groupName",
                 "default": null
             },
+            {
+                "key": "EnableLeftSidebar",
+                "display_name": "Display Notification Counters in Left Sidebar",
+                "type": "bool",
+                "help_text": "When false, the counters showing the user how many open/assigned issues they have in Github will not be shown in the Left Hand Sidebar on desktop browsers.",
```

```suggestion
                "help_text": "When false, the counters showing the user how many open/assigned issues they have in GitLab will not be shown in the Left Hand Sidebar.",
```

Leaving out "on desktop browsers" because it sounds like we only disable on desktop